File: 2017-06-01/Sensors/app/src/main/java/br/ufpe/cin/if1001/sensors/MainActivity.java
package br.ufpe.cin.if1001.sensors;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.GestureDetector;
import android.view.View;
import android.widget.Button;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button b1 = (Button) findViewById(R.id.btn1);
        b1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(getApplicationContext(),SensorManagerActivity.class));
            }
        });

        Button b2 = (Button) findViewById(R.id.btn2);
        b2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(getApplicationContext(),GestureDetectorActivity.class));
            }
        });

        Button b3 = (Button) findViewById(R.id.btn3);
        b3.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
            }
        });

    }
}

File: 2018.1/2018-04-16/ContentConsumer/app/src/main/java/br/ufpe/cin/if710/contentconsumer/MainActivity.kt
package br.ufpe.cin.if710.contentconsumer

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button

class MainActivity : Activity() {

    @Override
    protected fun onCreate(savedInstanceState: Bundle) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val b1 = findViewById(R.id.btn1) as Button
        b1.setOnClickListener(object : View.OnClickListener() {
            @Override
            fun onClick(v: View) {
                startActivity(Intent(getApplicationContext(), LerContatosArrayActivity::class.java))
            }
        })

        val b2 = findViewById(R.id.btn2) as Button
        b2.setOnClickListener(object : View.OnClickListener() {
            @Override
            fun onClick(v: View) {
                startActivity(Intent(getApplicationContext(), LerContatosCursorActivity::class.java))
            }
        })

        val b3 = findViewById(R.id.btn3) as Button
        b3.setOnClickListener(object : View.OnClickListener() {
            @Override
            fun onClick(v: View) {
                startActivity(Intent(getApplicationContext(), LerContatosLoaderActivity::class.java))
            }
        })

        val b4 = findViewById(R.id.btn4) as Button
        b4.setOnClickListener(object : View.OnClickListener() {
            @Override
            fun onClick(v: View) {
                startActivity(Intent(getApplicationContext(), ContentConsumerActivity::class.java))

            }
        })
        val b5 = findViewById(R.id.btn5) as Button
        b5.setOnClickListener(object : View.OnClickListener() {
            @Override
            fun onClick(v: View) {
                startActivity(Intent(getApplicationContext(), ContentConsumerSQLiteActivity::class.java))

            }
        })
    }
}
